content/en/platform/corda/5.0/developing-applications/packaging/code-signing/signing-cli.md

```diff
@@ -26,7 +26,7 @@ mycpb.cpb \
 --key "signing key 1"
 ```
 
-2. Supply the group policy into CPI version 2 by performing one of the following actions:
+2. Build the CPI. Supply the group policy into CPI version 2 by performing one of the following actions:
```

Comment:
```suggestion
2. Build a CPI (version 2). The group policy file can be passed into the CPI by either passing it to the cli package command parameters as a file or by piping it to the cli package command as shown below:
```

Reply:
Updated according to your suggestions @kyriathar